app/views/teachers/learning_resources/index.html.erb

```diff
@@ -1,20 +1,27 @@
-<h1>Learning resources</h1>
+<div class="container pt-5">
+  <h1 class="title">Learning resources</h1>
 
-<div id="teachers_learning_resources">
-  <% @teacher_learning_resources.each do |learning_resource| %>
-    <%= render "teachers/learning_resources/learning_resource", learning_resource: learning_resource%>
-    <p>
-      <%= link_to "Show this learning resource", teacher_learning_resource_path(@user, learning_resource) %>
-      <br>
-    </p>
-    <br>
-  <% end %>
-</div>
+  <div>
+    <% @teacher_learning_resources.each do |learning_resource| %>
+      <%= render "teachers/learning_resources/learning_resource", learning_resource: learning_resource%>
+    <% end %>
+  </div>
 
-<%= will_paginate @teacher_learning_resources, renderer: RemoteLinkPagination::LinkRenderer, :class=>"" %>
+  <%= will_paginate @teacher_learning_resources, renderer: RemoteLinkPagination::LinkRenderer, :class=>"" %>
```

Comment:
```suggestion
  <%= will_paginate @teacher_learning_resources, renderer: RemoteLinkPagination::LinkRenderer %>
```

Reply:
Same here. Actually `class` attr is totally useless, isn't it?